libtenzir/src/detail/process.cpp

```diff
@@ -75,6 +75,18 @@ static record get_status_proc() {
   return result;
 }
 
+static int64_t get_process_fds() {
```

Comment:
We have such functionality already in the code base: https://github.com/tenzir/tenzir/blob/main/libtenzir/src/os.cpp#L252

I don't think we want it twice.